Makefile

```diff
@@ -55,7 +55,7 @@ COMMON_ARGS += --build-arg=https_proxy=$(https_proxy)
 
 CI_ARGS ?=
 
-all: initramfs kernel installer talosctl talos
+all: initramfs kernel installer talosctl talosctl-image talos
```

Comment:
we need to add `talosctl-image` to the `push` target to make CI actually push the image

Reply:
Done.